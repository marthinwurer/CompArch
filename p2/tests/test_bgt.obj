000 2 00 00
002 2 ff ff
004 2 ff 7f
006 4 c1 1d f6 ff
00a 4 c2 1d f4 ff
00e 4 c3 1d f2 ff
012 2 03 00
014 2 c1 20
016 2 02 06
018 4 c0 65 01 00
01c 2 43 20
01e 2 01 06
020 2 02 01
022 4 c0 65 02 00
026 2 81 20
028 2 01 06
02a 2 02 01
02c 4 c0 65 04 00
030 2 42 20
032 2 02 06
034 4 c0 65 08 00
038 2 83 20
03a 2 01 06
03c 2 02 01
03e 4 c0 65 10 00
042 2 c2 20
044 2 02 06
046 4 c0 65 10 00
04a 2 41 20
04c 2 01 06
04e 2 02 01
050 4 c0 65 20 00
054 2 03 00
056 2 00 00
006
