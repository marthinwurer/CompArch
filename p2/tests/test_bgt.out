CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 6
000006:  PS[00]  016701 177766  MOV 177766(R7),R1
000012:  PS[04]  016702 177764  MOV 177764(R7),R2
000016:  PS[10]  016703 177762  MOV 177762(R7),R3
000022:  PS[00]  000003  BPT
  R0[000000]  R1[000000]  R2[177777]  R3[077777]
  R4[000000]  R5[000000]  R6[000000]  R7[000024]
000024:  PS[00]  020301  CMP R3,R1
000026:  PS[00]  003002  BGT 4
000034:  PS[00]  020103  CMP R1,R3
000036:  PS[11]  003001  BGT 2
000040:  PS[11]  000402  BR 4
000046:  PS[11]  020201  CMP R2,R1
000050:  PS[10]  003001  BGT 2
000052:  PS[10]  000402  BR 4
000060:  PS[10]  020102  CMP R1,R2
000062:  PS[01]  003002  BGT 4
000070:  PS[01]  020203  CMP R2,R3
000072:  PS[10]  003001  BGT 2
000074:  PS[10]  000402  BR 4
000102:  PS[10]  020302  CMP R3,R2
000104:  PS[13]  003002  BGT 4
000112:  PS[13]  020101  CMP R1,R1
000114:  PS[04]  003001  BGT 2
000116:  PS[04]  000402  BR 4
000124:  PS[04]  000003  BPT
  R0[000000]  R1[000000]  R2[177777]  R3[077777]
  R4[000000]  R5[000000]  R6[000000]  R7[000126]
000126:  PS[04]  000000  HALT

Machine Halted - HALT instruction

Simulated time 169 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
